docker-compose.yaml

```diff
@@ -7,8 +7,15 @@ services:
       - db
   start_applications:
```

Comment:
💡 We can split this task later into `start_check_legal_aid` and `start_case_handler_system` when we first get the need.

Reply:
Good shout!